libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -393,10 +418,17 @@ export function BmdPaperBallot({
     }
   }, [onRendered]);
 
+  const layout = find(
+    [...BMD_BALLOT_LAYOUTS[printType]].reverse(),
+    (l) => contests.length >= l.minContests
+  );
```

Comment:
Does this rely on `BMD_BALLOT_LAYOUTS` being specified in a certain order? Might be more defensive to just sort here before selecting a layout

Reply:
There's a unit test asserting the order of the layouts, but maybe this would be less confusing if we just explicitly sort here or add more obvious naming/documentation -- will update